pkg/postgres/pgutils/error.go

```diff
@@ -47,6 +49,14 @@ var transientPGCodes = set.NewFrozenStringSet(
 	"58030", // io_error
 )
 
+// IsQueryTimeoutError specifies if the error is a statement timeout error
```

Comment:
super-nit, or should the func be public:
```suggestion
// isQueryTimeoutError specifies if the error is a statement timeout error
```